instrumentation/events/lttng-module/lttng-statedump.h

```diff
@@ -136,6 +136,7 @@ LTTNG_TRACEPOINT_EVENT(lttng_statedump_interrupt,
 		ctf_integer(unsigned int, irq, irq)
 		ctf_string(name, chip_name)
 		ctf_string(action, action->name ? : "")
+		ctf_integer(pid_t, thread, action->thread != NULL ? action->thread->pid : 0)
```

Comment:
I am tempted to rename the field "thread" by "tid" here, to stay consistent with the rest of lttng-modules. I will do it and merge.